xml/cl.xml

```diff
@@ -1848,7 +1848,7 @@ server's OpenCL/api-docs repository.
         <enum value="0x2051"        name="CL_MEM_DEVICE_HANDLE_LIST_KHR"/>
         <enum value="0x2052"        name="CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEAR_IMAGES_HANDLE_TYPES_KHR"/>
         <enum value="0x2053"        name="CL_SEMAPHORE_DEVICE_HANDLE_LIST_KHR"/>
-            <unused start="0x2054" end="0x2054"/>
+        <enum value="0x2054"        name="CL_SEMAPHORE_IS_EXPORTABLE_KHR"/>
```

Comment:
Can you please add this enum to the "require" section for cl_khr_semaphore (or whatever extension we decide to make this enum a part of)? This is needed so headers generate properly.

This will probably be around [here](https://github.com/KhronosGroup/OpenCL-Docs/blob/main/xml/cl.xml#L6945).

Reply:
Discussed in the November 14th teleconference: should this also be `CL_SEMAPHORE_EXPORTABLE` rather than `CL_SEMAPHORE_IS_EXPORTABLE`?  This seems to be more consistent with other OpenCL enum names.

Reply:
Done